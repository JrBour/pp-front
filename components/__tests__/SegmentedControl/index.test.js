import { shallowMount } from '@vue/test-utils'
import SegmentedControl from '@/components/SegmentedControl'

describe('SegmentedControl', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallowMount(SegmentedControl, {
      propsData: {
          first: 'first',
          second: 'second',
      },
    })
  })

  describe('methods', () => {
    it('should emitted removeParticipants methods', async () => {
      await wrapper.findAll('button').at(0).trigger('click')
      expect(wrapper.emitted('get-segment')).toBeTruthy()
    })
  })

  describe('snapshot', () => {
    it('renders properly', () => {
      expect(wrapper.html()).toMatchSnapshot()
    })
  })
})
