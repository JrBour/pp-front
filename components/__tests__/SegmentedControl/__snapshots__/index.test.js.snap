// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SegmentedControl snapshot renders properly 1`] = `
<div class="segmented"><button class="active">
    first
  </button> <button class="">
    second
  </button></div>
`;
